render/fx_renderer/fx_pass.c

```diff
@@ -859,6 +982,40 @@ static struct fx_framebuffer *get_main_buffer_blur(struct fx_gles_render_pass *p
 	// Artifacts with NEAREST filter
 	fx_options->tex_options.base.filter_mode = WLR_SCALE_FILTER_BILINEAR;
 
+	TRACY_BOTH_ZONES_START(renderer);
+	TRACY_ZONE_TEXT_f("dst_box (WxH, X, Y): %ix%i, %i, %i",
```

Comment:
this goes for all tracy functions, but why use %i over %d? is this something specific to tracy? (I know scanf for example treats %i different than %d)